README.md

```diff
@@ -1,14 +1,103 @@
+| ```ACTk```        | CMB convergence                   | [Darwish et al, 2020](https://arxiv.org/abs/2004.01139)               | [catalogue](https://lambda.gsfc.nasa.gov/product/act/act_dr4_derived_maps_get.html)               | J. Ruiz-Zapatero       |
+| ```BOSS```        | Galaxy clustering                 | [Alam et al, 2016](https://arxiv.org/abs/1607.03155)                  | [catalogue](https://data.sdss.org/sas/dr12/boss/)                                                 | J. Ruiz-Zapatero       |
+| ```CIBLenz```     | CIB ansitropies                   | [Lenz et al, 2019](https://arxiv.org/abs/1905.00426)                  | [catalogue](https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/8A1SR3)      | J. Ruiz-Zapatero       |
+| ```CatWISE```     | Galaxy clustering                 | [Marocco et al, 2020](https://arxiv.org/abs/2012.13084)               | [catalogue](https://irsa.ipac.caltech.edu/data/WISE/CatWISE/gator_docs/catwise_colDescriptions.html) | F. Oliveira-Franco  |
+| ```DELS```        | Galaxy clustering                 | [Dey et al, 2019](https://arxiv.org/abs/1804.08657)                   | [catalogue](https://www.legacysurvey.org/dr8/catalogs/)                                           | D. Alonso              |
+| ```DESY1gc```     | Galaxy clustering                 | [Abbot et al, 2017](https://arxiv.org/abs/1708.01530)                 | [catalogue](https://desdr-server.ncsa.illinois.edu/despublic/y1a1_files/redmagic/)                | C. Garcia-Garcia       |
+| ```DESY1wl```     | Weak lensing                      | [Abbot et al, 2017](https://arxiv.org/abs/1708.01530)                 | [catalogue](https://desdr-server.ncsa.illinois.edu/despublic/y1a1_files/shear_catalogs/)          | C. Garcia-Garcia       |
+| ```DESY3wl```     | Weak lensing                      | [Abbot et al, 2021](https://arxiv.org/abs/2105.13549)                 | [catalogue](https://desdr-server.ncsa.illinois.edu/despublic/y3a2_files/)                         | C. Garcia-Garcia       |
+| ```eBOSS```       | Quasar clustering                 | [Gil-Marin et al, 2018](https://arxiv.org/abs/1801.02689)             | [cataloge](https://data.sdss.org/sas/dr14/eboss/)                                                 | J. Ruiz-Zapatero       |
+| ```HSC_DR1wl```   | Weak lensing                      | [Aihara et al, 2017](https://arxiv.org/abs/1702.08449)                | [catalogue](https://hsc-release.mtk.nao.ac.jp/doc/index.php/tools/)                               | D. Alonso              |
+| ```KiDS1000```    | Weak lensing                      | [Heymans et al, 2020](https://arxiv.org/abs/2007.15632)               | [catalogue](https://kids.strw.leidenuniv.nl/DR4/data_files/)                                      | D. Alonso              |
```

Comment:
Maybe we should cite this one: https://arxiv.org/pdf/2007.15633.pdf?

Reply:
or this one? https://arxiv.org/pdf/2007.01845.pdf

Reply:
Cited the catalog paper instead